servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/AbstractTestRest.java

```diff
@@ -940,6 +940,122 @@ public void filterEntriesByNamespace() throws NessieConflictException, NessieNot
     tree.deleteBranch(branch, tree.getReferenceByName(branch).getHash());
   }
 
+  @Test
+  public void filterEntriesByNamespaceAndPrefixDepth()
+      throws NessieConflictException, NessieNotFoundException {
+    final String branch = "filterEntriesByNamespaceAndPrefixDepth";
+    Reference r = tree.createReference("main", Branch.of(branch, null));
+    ContentsKey first = ContentsKey.of("a", "b", "c", "firstTable");
+    ContentsKey second = ContentsKey.of("a", "b", "c", "secondTable");
+    ContentsKey third = ContentsKey.of("a", "thirdTable");
+    ContentsKey fourth = ContentsKey.of("a", "b", "fourthTable");
+    ContentsKey fifth = ContentsKey.of("a", "boo", "fifthTable");
+    tree.commitMultipleOperations(
```

Comment:
Better put those into a `for (int i = 1; i <= 5; i++)` loop (less code)

Reply:
done